framework/test/jest/integration/specs/modules/chain/components/storage/entities/block_temp.spec.js

```diff
@@ -234,4 +234,20 @@ describe('TempBlock', () => {
 			await expect(TempBlockEntity.delete({ height })).resolves.toBeNull();
 		});
 	});
+
+	describe('truncate', () => {
+		it('should truncate all rows from table', async () => {
```

Comment:
don't we need a setup for this?

Reply:
Before every test, a table gets created with two transactions in. Should be good! :)